docs/contributing/continuous-integration.rst

```diff
@@ -37,3 +59,10 @@ Attention and care must be given to merging PRs that require other Pull Requests
 all required PRs should be ready to merge--meaning that all tests/checks should be passing, the code
 review requirements should be met, etc. When merging, the PR along with its required PRs should all
 be merged at the same time. This is necessary to ensure that test breakages don't block other PRs.
+
+Contributing to tests
+----------------------
+
+Pulp team will never release a new Pulp version with failing units or
```

Comment:
A new version of Pulp will only be released when all unit and functional tests are passing. Contributing tests is a great way to ensure that your workflows never regress.